1-getting-started.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `1-getting-started.ipynb` Cell 85 Line 17_
```markdown
```
$conda create -n py4wrds python=3.12
$conda activate py4wrds
```

#### Install Key Packages

`conda install numpy pandas matplotlib`
```
Because the py4wrds conda environment will already contain these packages, this command won't install anything. (Which might still be fine as an example! Or consider installing some random small python package that we won't use for the course)